### DISCUSSION TITLE
Typing Input not working when remoting WhatsApp application

### DISCUSSION THREAD

**Discussion by Lim1029** [kimmknight/remoteapptool#149] [Q&A]
I encountered an issue while using a RemoteApp tool to access the WhatsApp Desktop application installed on a host PC. While I can use the mouse to interact with the interface (e.g., clicking on buttons or text fields), typing in text fields (such as the message search bar or message input box) does not work when using the RemoteApp.

WhatsApp Installation Path:
C:\Program Files\WindowsApps\5319275A.WhatsAppDesktop_2.2447.5.0_x64__cv1g1gvanyjgm\WhatsApp.exe

![{A70953F8-D914-4E4D-99BF-F11393A213AA}](https://github.com/user-attachments/assets/1099153e-a4ef-4907-b1c1-d3a65c8d1b33)

**Comment by kimmknight** [comment]
As you are connecting to the Remote app, this is not an issue with RemoteApp Tool - it is likely an issue with RemoteApp itself.

I will convert the issue to a discussion post. Hopefully someone might have an answer or suggestion.
